src/k8s-engine/lima.ts

```diff
@@ -36,6 +38,7 @@ import SERVICE_BUILDKITD_CONF from '@/assets/scripts/buildkit.confd';
 import mainEvents from '@/main/mainEvents';
 import UnixlikeIntegrations from '@/k8s-engine/unixlikeIntegrations';
 import { getImageProcessor } from '@/k8s-engine/images/imageFactory';
+import { KubeClient } from '~/k8s-engine/client';
```

Comment:
```suggestion
import { KubeClient } from '@/k8s-engine/client';
```
(I assume this was done automatically by the editor)

Reply:
yup, it's a jetbrains thing